packages/@react-aria/color/docs/useColorArea.mdx

```diff
@@ -0,0 +1,584 @@
+If no `xChannel` or `yChannel` is provided, for the RGB color space, the `red` color channel maps to the horizontal axis or `xChannel`,
+and the `green` color channel maps to the vertical axis or `yChannel`. Similarly, for the HSL and HSB color spaces, the `hue` color
+channel maps to the horizontal axis or `xChannel`, and the `saturation` color channel maps to the vertical axis or `yChannel`.
+
+```tsx example
+<label id="hsb-label-id">x: Hue, y: Saturation</label>
+<ColorArea
+  aria-labelledby="hsb-label-id"
+  defaultValue={'hsb(219, 58%, 93%)'} />
+```
+
+### Controlled
+
+A color area can be made controlled using the `value` prop. The <TypeLink links={statelyDocs.links} type={statelyDocs.exports.parseColor} />
+function is used to parse the initial color from an RGB, HSL or HSB string, stored in state. The `onChange` prop
+is used to update the value in the state when the user drags the thumb. This is the more common usage, because it allows to adjust the third
+color channel using a separate control, like a color slider using the [useColorSlider](useColorSlider.html) hook or a color wheel using the
+[useColorWheel](useColorWheel.html) hook.
+
+```tsx example
+import {parseColor} from '@react-stately/color';
+
+function Example() {
+  let [
+    color,
+    setColor
+  ] = React.useState(parseColor('hsb(219, 58%, 93%)'));
+  let [
+    xChannel,
+    yChannel,
+    zChannel
+  ] = color.getColorChannels();
+  return (
+    <>
+      <label id="hsb-label-id-1">x: {color.getChannelName(xChannel, 'en-US')}, y: {color.getChannelName(yChannel, 'en-US')}</label>
+      <ColorArea
+        aria-labelledby="hsb-label-id-1"
+        value={color}
+        onChange={setColor}
+        xChannel={xChannel}
+        yChannel={yChannel} />
+      <label>
+        {color.getChannelName(zChannel, 'en-US')}<br/>
+        <input
+          type="range"
+          min={color.getChannelRange(zChannel).minValue}
+          max={color.getChannelRange(zChannel).maxValue}
+          value={color.getChannelValue(zChannel)}
+          onChange={e => setColor(color.withChannelValue(zChannel, e.target.value))}
+          style={{
+            width: 192
+          }} />
+      </label>
+      <p>Current color value: {color.toString('hsb')}</p>
+    </>
+  );
+}
+```
+
+### onChangeEnd
+
+The `onChangeEnd` prop can be used to handle when a user stops dragging the color area, whereas the `onChange`
+prop is called as the user drags.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('hsl(0, 100%, 50%)'));
+  let [hueChannel, saturationChannel, lightnessChannel] = color.getColorChannels();
+  return (
+    <>
+      <label id="slh-label-id-2">x: {color.getChannelName(saturationChannel, 'en-US')}, y: {color.getChannelName(lightnessChannel, 'en-US')}</label>
+      <ColorArea
+        aria-labelledby="slh-label-id-2"
+        value={color}
+        onChange={setColor}
+        onChangeEnd={setColor}
```

Comment:
I tried that originally, but without `onChange`, the thumb no longer drags, because the value is now controlled. Arrow keys will work, because both `onChange` and `onChangeEnd` fire with each keyboard event. But with only `onChangeEnd`, the value stored in state does not update when the user drags the handle, so it does not move.

Reply:
Can we update a little swatch sibling component onChangeEnd? otherwise this section does feel a bit strange